### PR TITLE
Add Medellín chapter to README

### PR DESCRIPTION
**Pull request by c4milo**
We had our first meetup 5 days ago: https://www.meetup.com/paperswelovemde/events/238392836/

Some pictures can be seen in our Twitter account: https://twitter.com/paperswelovemde \o/ 

Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ Here are our official chapters. Let us know if you are interested in [starting o
 * [London](http://www.meetup.com/papers-we-love-london)
 * [Los Angeles](http://www.meetup.com/papers-we-love-la)
 * [Madrid](http://www.meetup.com/Papers-We-Love-Madrid/)
+* [Medellín](https://www.meetup.com/paperswelovemde/events/238392836/)
```
Comment (zeeshanlakhani) [inline README.md]: Can you just add the the main *meetup* link (not the event itself)? Thanks!

**Comment by zeeshanlakhani** [comment]
minor comment @c4milo. Please fix and we'll merge it.

**Comment by c4milo** [comment]
@zeeshanlakhani done! Thanks!

**Comment by NewAlexandria** [comment]
Cheers to your new chapter 🎂

**Comment by c4milo** [comment]
Thanks you @NewAlexandria!

**Comment by hakutsuru** [comment]
+1

**Pull request merged**